.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,7 @@
 		"ghcr.io/devcontainers/features/python:1": {},
 		"ghcr.io/devcontainers/features/node:1": {
 			"nodeGypDependencies": true,
-			"version": "19.7.0"
```

Comment:
question: what's the version downgrade for?

Reply:
we are currently restricted from using 19 due to this change: https://github.com/determined-ai/determined/pull/6759